content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -0,0 +1,245 @@
+configurações de armazenamento, plugins de dispositivos, kube-proxy, e outros
+componentes para nós do Windows, sem a necessidade de *proxies* dedicados ou
+a instalação direta dos serviços de host.
+
+Tarefas administrativas, como a instalação de `patches` de segurança, 
+coleta de logs de eventos, e mais pode ser realizado sem a necessidade 
+de operadores de cluster, que façam login em cada nó do Windows. 
+Contêineres `HostProcess` podem ser executados como qualquer usuário 
+que esteja disponível no host ou esteja no domínio da máquina host, 
+permitindo aos administradores restringirem o acesso aos recursos, 
+por meio de permissões de usuário. Enquanto nenhum sistema de arquivos 
+ou isolamento de processos são suportados, um novo volume é criado no host 
+ao iniciar o contêiner para fornecer um espaço de trabalho limpo e consolidado. 
+Contêineres `HostProcess` também podem ser construídos sobre as imagens base 
+do Windows existente, e não herdam os mesmos
+[requisitos de compatibilidade](https://docs.microsoft.com/virtualization/windowscontainers/deploy-containers/version-compatibility)
+como contêineres do `Windows Server`, o que significa que a versão das 
+imagens base não precisa combinar com a do host. É, no entanto, 
+recomendado que você use a mesma versão da imagem base como seu
+servidor Windows de cargas de trabalho, para garantir que você não tenha 
+nenhuma imagem não utilizada ocupando espaço no nó. Contêineres HostProcess 
+também suportam [montagem de volume](#volume-mounts) dentro do volume do contêiner.
+
+### Quando devo usar um contêiner do Windows HostProcess?
+
+- Quando você precisar executar tarefas que exigem a rede do namespace no host.
+Contêineres HostProcess tem acesso às interfaces de rede e endereços IP do host.
+- Você precisa acessar recursos no host como o sistema de arquivos, 
+os logs de eventos, etc.
+- Instalação de drivers de dispositivo específicos ou serviços Windows.
+- Consolidação de tarefas administrativas e políticas de segurança. 
+Isso reduz o grau de privilégios necessários para os nós do Windows.
+
+
+## {{% heading "prerequisites" %}}
+
+<!-- change this when graduating to stable -->
+
+Este guia de tarefas é específica para o Kubernetes v{{< skew currentVersion >}}.
+Se você não está executando o Kubernetes v{{< skew currentVersion >}}, 
+verifique a documentação para essa versão do Kubernetes.
+
+No Kubernetes {{< skew currentVersion >}}, O recurso de contêiner do HostProcess 
+é ativado por padrão. 
+O kubelet irá comunicar-se com o contêiner diretamente passando a flag HostProcess 
+via `CRI`. Você pode usar a versão mais recente do containerd (v1.6+) para executar 
+contêineres HostProcess.
+[Como instalar o containerd.](/docs/setup/production-environment/container-runtimes/#containerd)
+
+Para *desabilitar* contêineres HostProcess você precisa passar a seguinte flag 
+de portal de funcionalidade para o **kubelet** e **kube-apiserver**:
```

Comment:
Since "feature gate" is a Kubernetes technical term, it would be better to not translate it and instead stylize it as italic, e.g. "você precisa passar o seguinte _feature gate_...".